src/hmm/transition-model.h

```diff
@@ -288,8 +288,8 @@ class TransitionModel {
 
   HmmTopology topo_;
 
-  /// Triples indexed by transition state minus one;
-  /// the triples are in sorted order which allows us to do the reverse mapping from
+  /// Tuples indexed by transition state minus one;
+  /// the tuples are in sorted order which allows us to do the reverse mapping from
   /// triple to transition state
```

Comment:
there is one  more 'triple' on this line.

Reply:
yeal, I fixed it.